pkg/skaffold/build/buildpacks/lifecycle.go

```diff
@@ -83,6 +85,16 @@ func (b *Builder) build(ctx context.Context, out io.Writer, a *latest.Artifact,
 }
 
 func runPackBuild(ctx context.Context, out io.Writer, localDocker docker.LocalDaemon, opts pack.BuildOptions) error {
+	type notATerminal struct {
+		io.Writer
+	}
+
+	// If out is not a terminal, let's make sure pack doesn't output with colors
+	if _, isTerm := util.IsTerminal(out); !isTerm {
+		color.Disable(true)
```

Comment:
I'm guessing that `pack` uses `heroku/color` under the hood?  Would be worth a comment here?

Reply:
ok